guerravc/KChallenge
# Feature two

#### Purpose

#### Details
Now we added a segmented control where you can choose between see Most Popular Movies or Now playing Movies.

#### Steps to Test
* Run the application.
* Choose Now Playing Movies.
* Now can you see the movies that are playing now
